package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "comparative-parquet",
-  "version": "0.4.0",
+  "version": "0.5.0",
```

Comment:
You know what let's publish a `1.0.0`.

Reply:
If we're publishing a 1.0.0 can I drop the backwards compatibility enum values? They throw compiler warnings because they're technically not actual values in the real enum and just sort of tacked on on the JS side.